src/main/java/com/google/devtools/build/lib/rules/cpp/CppCompilationContext.java

```diff
@@ -48,11 +49,11 @@
   public static final CppCompilationContext EMPTY = new Builder(null).build();
 
   private final CommandLineContext commandLineContext;
-  
+
```

Comment:
Remove added blank lines (I like them there too, but in theory they'd mess up blame information and make rollbacks harder).

Reply:
Will do - this may have came from some other change